sway/tree/container.c

```diff
@@ -658,18 +717,82 @@ void container_calculate_title_height(struct sway_container *container) {
 	container->title_height = height;
 }
 
+/**
+ * Calculate and return the length of the concatenated child titles.
+ * An example concatenated title is: V[Terminal, Firefox]
+ * If buffer is not NULL, also populate the buffer with the concatenated title.
+ */
+static size_t concatenate_child_titles(struct sway_container *parent,
```

Comment:
Hum, is that how i3 does it? Titles could end up being veeery long

Reply:
I just checked and i3 uses the window class. I've changed it to use the class too, with fallback to app_id. Also, they don't use commas so I've removed those.